projects/core/src/main/resources/data/computercraft/lua/rom/apis/colors.lua

```diff
@@ -363,3 +363,20 @@ function toBlit(color)
     return color_hex_lookup[color] or
         string.format("%x", math.floor(math.log(color) / math.log(2)))
 end
+
+--- Converts the given paint/blit hex character (0-9a-f) to a color.
+--
+-- This is equivalent to converting the hex character to decimal and then 2 ^ decimal
+--
+-- @tparam string hex The paint/blit hex character to convert
+-- @treturn number The color
+
```

Comment:
Can you add the version here too?

```suggestion
-- @since 1.105.0
```

Might be nice to have an example here too, but I think that's less important.